static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -166,21 +168,21 @@ function HoverHeader({
 }
 
 type IssueContextProps = {
-  api: Client;
   dataRow: EventData;
-  eventID?: string;
+  organization?: Organization;
 };
 
 function IssueContext(props: IssueContextProps) {
   const statusTitle = t('Issue Status');
+  const {dataRow, organization = null} = props;
```

Comment:
Can we `useOrganization` here and for the other contexts? I feel like this can be a required prop as well

Reply:
Also, can this just use `BaseContextProps`?

Reply:
Yup you're correct. Used BaseContextProp and made organization required.